src/Interface/Modules/Render/ViewSceneManager.cc

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
@Haydelj Yeah looks like it's this h/cc pair, and ViewScene/SRInterface changes, the latter set needs a redo. But a rebase ought to make it easier to start merging.